cookbooks/bcpc-hadoop/attributes/hdfs.rb

```diff
@@ -13,10 +13,14 @@
 # # set to nil to calculate dynamically based on available memory
 default['bcpc']['hadoop']['namenode']['xmn']['max_size'] = 128
 default['bcpc']['hadoop']['namenode']['xmx']['max_ratio'] = 0.25
-default['bcpc']['hadoop']['namenode']['jmx']['port'] = 10111
+default['bcpc']['hadoop']['namenode']['jmx']['port'] = 10_111
```

Comment:
I don't think we should be using JMX port anymore? (We use jmxtrans_agent to send data so we don't need to network poll or expose JMX.)

Reply:
I tried to please rubocop here. My change is just to add `_` in a number.